src/studio-home/data/api.js

```diff
@@ -40,6 +40,11 @@ export async function getStudioHomeLibraries() {
   return camelCaseObject(data);
 }
 
+export async function getStudioHomeLibrariesV2() {
```

Comment:
This function should define the type of the returned data.

Reply:
Will do.